cryptographic_estimators/SDEstimator/sd_problem.py

```diff
@@ -47,53 +44,56 @@ def __init__(self, n: int, k: int, w: int, **kwargs):
         self.parameters[SD_CODE_DIMENSION] = k
         self.parameters[SD_ERROR_WEIGHT] = w
 
-        self.nsolutions = kwargs.get("nsolutions", max(
-            self.expected_number_solutions(), 0))
+        self.nsolutions = kwargs.get("nsolutions", max(self.expected_number_solutions(), 0))
 
     def to_bitcomplexity_time(self, basic_operations: float):
         """
-        Returns the bit-complexity corresponding to basic_operations field additions
+        Calculates the bit-complexity corresponding to the number of field additions.
```

Comment:
I think adding:
> , which are the `basic_operations` for SDAlgorithms

should be helpful. As we often refer to `basic_operations` in the help files